### PR TITLE
Feature: node detail display in topo circle-chart view.

### PR DESCRIPTION
**Pull request by bigflybrother**

### CODE REVIEW

**Comment by wu-sheng** [comment]
Screenshot?

**Comment by bigflybrother** [comment]
Add the circle-chart's node click event to show current node informations.

641572336723_ pic_hd

File: vue.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/graphql': {
-        target: 'http://127.0.0.1:12800',
+        target: 'http://122.112.182.72:8080',
```
Comment (JaredTan95) [inline vue.config.js]: how about revert it?

Comment (wu-sheng) [reply vue.config.js]: Yes. This should not be changed.

**Comment by TinyAllen** [comment]
截屏2019-10-30上午10 50 07
In this way , the underside panel has a gap cannot keep out

**Comment by wu-sheng** [comment]
@bigflybrother Could you submit a sync PR to main repo?

**Pull request merged**